free-2.md

```diff
@@ -11,41 +11,82 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, the competetive Pokémon meta has grown very stale, with the same
+pokémon on each team. I think that this is partially due to the complexity of
+differnt builds and values leading to such a huge variety of thigns that might
+work, that players find it hard to find new viable builds. In addition, testing
+new theories is a very time consuming process. Hopefully, this could speed up
+testing.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A language to specify the moves and values of a pokémon could be very useful as
```

Comment:
A DSL definitely makes sense for this and I feel like it'd be a great way to practice language design, by creating a vocabulary for the different moves and values and designing methods for combining them.